solutions/cses-1161.mdx

```diff
@@ -20,13 +20,14 @@ If you're wondering why Huffman Coding always produces an optimal solution, see
 
 ## Implementation
 
+As we want to selete both shortest sticks and then insert the combined new stick, we will want to have a data structure where we can get the smallest value in the collection and add new value to it. The insert operation can be done easily with a priority queue in $\mathcal{O}(\log n)$ time while getting the smallest value takes constant time.
+
+This gives us a time complexity of $\mathcal{O}(n \log n)$ in total. 
```

Comment:
maybe better to do this as **Time Complexity** $\mathcal{O}(n \log n)$ under the ##Implementation to be consistent with the other solutions?